superset/reports/commands/exceptions.py

```diff
@@ -163,10 +165,12 @@ class ReportScheduleExecuteUnexpectedError(CommandException):
 
 
 class ReportSchedulePreviousWorkingError(CommandException):
+    status = 429
```

Comment:
nit: idk if this is a too many request fit here but i also don't what other code fits

Reply:
Yeah, I wrote the same thing in the summary. I feel like ideally this wouldn't be an error.

Reply:
I don't think this will ever generate an HTTP response, this is only used on `execute.py` that always run on the workers

Reply:
Thanks @dpgaspar. The goal is to help log the user errors vs system errors more granularly. Right now they are all getting logged as one event. Do you think that the status codes is a viable approach for logging purposes?

Reply:
Thanks @dpgaspar! @john-bodley, @betodealmeida and I were talking about `error_type`s as well and concluded that errors should not be raised, only exceptions, and that we can include errors with an exception, but not raise them. Do you have thoughts on that @dpgaspar?